Tools/Linux_HAL_Essentials/pru/aiopru/RcAioPRU.p

```diff
@@ -387,9 +396,13 @@ mainloop:
    RCOUT_PWM RC_CH_4_PIN, register.ch_4_next_time, RC_CH_4_ENABLE, CH_4_PULSE_TIME_RAM_OFFSET, CH_4_T_TIME_RAM_OFFSET
    RCOUT_PWM RC_CH_5_PIN, register.ch_5_next_time, RC_CH_5_ENABLE, CH_5_PULSE_TIME_RAM_OFFSET, CH_5_T_TIME_RAM_OFFSET
    RCOUT_PWM RC_CH_6_PIN, register.ch_6_next_time, RC_CH_6_ENABLE, CH_6_PULSE_TIME_RAM_OFFSET, CH_6_T_TIME_RAM_OFFSET
+#ifdef BBBLUE
```

Comment:
Not my business to meddle in this, but this would look nicer to me without the line repetition below; you can use `#if defined(BBBLUE) || defined(BBBMINI)` here and avoid the repetition below.

Reply:
@OXINARF The pasm assembler can unfortunately not handle this syntax.

Reply:
Oh really? Interesting. Always learning 🙂